src/componentTest/resources/expectedTemplateContent/add-provider-pact-spec/src/componentTest/groovy/com/blackbaud/someservice/pact/ProviderVerificationSpec.groovy

```diff
@@ -0,0 +1,48 @@
+package com.blackbaud.someservice.pact
+
+import com.blackbaud.pact.BaseAuthPactProviderSpecification
+import com.blackbaud.pact.api.InteractionDetails
+import com.blackbaud.pact.support.ProviderPactInitializer
+import com.blackbaud.pact.support.VerificationValidator
+import com.blackbaud.testsupport.BBAuthSupport
+import org.springframework.beans.factory.annotation.Autowired
+import spock.lang.Unroll
+
+@ComponentTest
+class ProviderVerificationSpec extends BaseAuthPactProviderSpecification implements BBAuthSupport {
+
+    @Autowired
+    ProviderStateInitializer providerStateInitializer
+
+    @Unroll
+    def "#interactionDetails.type #interactionDetails.consumer.name #interactionDetails.interaction.description"() {
+        given:
+        this.providerPactExecutor.initialize(interactionDetails)
+
+        and:
+        initializeMocks(interactionDetails)
+
+        when:
+        VerificationValidator validator = this.providerPactExecutor.execute(interactionDetails)
+
+        then:
+        validator.validateResults()
+
+        where:
+        interactionDetails << pactInteractions
+    }
+
+    void initializeMocks(InteractionDetails interactionDetails) {
+        // initialize any required mocks
+    }
+
+    @Override
+    String getPactServiceName() {
+        return 'some-service'
+    }
+
+    @Override
+    ProviderPactInitializer getInitializer() {
+        return this.providerStateInitializer
+    }
```

Comment:
in order to test with a local json, devs need to override this method 
```
@Override
  String getLocalPath() {
    return null // use a full path if you want to run your test with local interactions
  }
``` 
I've had several people ask me about the documentation for this method. What do you think about adding that method into the template? It is optional so it's weird to have it by default. But it will be a very convenient way to tell devs that this functionality exists. Thoughts?

Reply:
makes sense to me, i'll add that